docs/update-strategies.md

```diff
@@ -1,6 +1,7 @@
 ## This doc is still under development
 
-* `all` strategy uses `version` passed to options as a cache tag. When version changes, old version cache is removed and new files are downloaded. Of course if files has same name were not changed (304 status returned), then probably browser won't download them again and will just update cache.
-* `changed` strategy is more advanced than `all` or `hash`. To work properly it requires output assets to have unique names between compilations, e.g. _file's unique hash_ in its name. **With this strategy enabled, `index.html` file (or other files without dynamic name) should be placed in `main` section of the cache, otherwise they won't be revalidated**.
+* **`all`** strategy uses `version` passed to `options` as a cache tag. When version changes, old version cache is removed and new files are downloaded. Of course if files with the same name were not changed (HTTP status 304), the browser probably won't download them again and will just update cache.
+* **`changed`** strategy is more advanced than `all` or `hash`. `offline-plugin` will calculate the files' hashes on it's own, independant of `webpack`. Even if a file is renamed, if the hash is the same, the file isn't redownloaded, but rather just renamed. **With this strategy enabled, `index.html` (or other files without dynamic name) should be placed in `main` section of the cache, otherwise they won't be revalidated**.
   * For `ServiceWorker` this means that only new files will be downloaded and missing files deleted from the cache.
-  * For `AppCache` it's basically same as previous strategies since `AppCache` revalidates all the assets. 304 HTTP status rule of course still works.
\ No newline at end of file
+  * For `AppCache` it's basically the same as previous strategies since `AppCache` revalidates all the assets. (HTTP status 304 rule still applies).
+* `updateStrategy` does not apply to externals, as they're always revalidated/redownloaded on SW update.
```

Comment:
👍
Make it `externals` though and `ServiceWorker` instead of SW.
(Yeah, I know about `ServiceWorker` and `AppCache` vs. `option`. I'm just weird..)

Reply:
fixed